CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Bugfixes
 - [#1971](https://github.com/influxdb/influxdb/pull/1971): Fix leader id initialization.
+- [#1975](https://github.com/influxdb/influxdb/pull/1975): Require parameter for query endpoint.
```

Comment:
This should be a bit more precise. Which parameter after all? "Require 'q' parameter for query endpoint" perhaps?